source/reporting/Report database.rst

```diff
@@ -0,0 +1,248 @@
+
+[dbo].[ProductReportingGroups]
+------------------------------
+ProductReportingGroups are used to put products into different groups. For example, it can be used to make it clear that products A and B belong to report group 1, while products C, D and E belong to report group 2.
+
+
+[dbo].[Products]
+----------------
+Products provides an overview of all products that are available, and has a more detailed description of these.
+
+Special fields in Products:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**Productcategory**: This is a grouping table for the report layer, which can be used to split the reporting into groups. You create a group, and then link products to the groups.
+
+ 
+
+[dbo].[ReportingGroups]
+-----------------------
+Report groups are defined to distinguish between different product types. Examples of this could be that one product is digital, another is on paper (physical), while a third is a combination of both digital and paper.
+ 
+
+[dbo].[SubscriberContacts]
+------------------------
+All contact addresses: Main addresses, billing addresses and temporary addresses.
+
+Special fields in SubscriberContacts:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**IsPrimary**: Value 1 means the address is home address, 0 is billing address or temporary address.
+
+**Identification**: May contain information about organization number, customer reference and more. JSON functions must be used to retrieve each individual value in this type of field.
+ 
+
+[dbo].[Subscribers]
+-------------------
+Subscriber is a register of all subscribers and contains all contact information. This information is used for deliveries and invoicing. Examples of use are sending a PDF invoice to an e-mail address or sending a physical invoice to a postal address.
+
+Special fields in Subscribers:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**ContactId**: This is a reference to the subscriber contact from which the contact information is retrieved.
+
+**ExternalId**: External customer number of a third party system.
+
+ 
+[DBO].[Tag]
+---------
+Free text that can be sent with an order, but this is used differently from customer to customer. The idea is to be able to connect to other tables to be able to retrieve information. Information in Tags can also be created manually.
+
+When you send a tag with an order, two tags are currently created in the report layer. One with type Subscriber and one with type SubscriberAccount. Referenceid then correspond henholdsvi s SubscriberId and SubscriberAccount . Thus you can search for all subscribers who have a relationship with for example ShopID = 458. The vast majority of tables in the report layer have a connection to subscriberide and can then be grouped / filtered in relation to Tag.
+ 
+
+[economy].[AccountTransaction]
+----------------------------
+Saldo transactions can, among other things, be used for purchases from agreements. For example, if you agree on a payment claim, which you will later buy out of, then this table will be used.
```

Comment:
I think it's better described as balance-transactions of different types. Will be created if paid too much or too litle, if a subscription is cancelled or for imported transactions.

Reply:
Updated and this example part is now removed: "For example, if you agree on a payment claim, which you will later buy out of, then this table will be used."